keyboards/hineybush/hineyg80/hineyg80.h

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2019 %YOUR_NAME%
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+/* Keymap _BL: (Base Layer) Default Layer
+   * ,------------------------------------------------------.      ,-------------------.
+   * |00 | 10 | 01 | 11 | 02 | 12 | 03 | 13 | 04| 14|05|15|06|     |07  |17  |08  |18  |
+   * `-------------------------------------------------------'     |-------------------|
+   *                                                               |27  |16  |28  |38  |
+   * ,-----------------------------------------------------------. |-------------------|
+   * |20 |30 |21 |31 | 22| 32| 23| 33| 24| 34| 25|35 |26 |36 |37 | |47  |57  |48  |58  |
+   * |-----------------------------------------------------------| |-------------------|
+   * |40   |50 |41 |51 |42 |52 |43 |53 |44 |54 |45 |55 |46 | 56  | |67  |77  |68  |78  |
+   * |-----------------------------------------------------------| |-------------------|
+   * |60    |70 |61 |71 |62 |72 |63 |73 |64 |74 |65 |75 |66 | 76 | |87  |97  |88  |98  |
+   * |-----------------------------------------------------------' |-------------------|
+   * |80 |90  |81 |91 |82 |92 |83 |93 |84 |94 |85 | 95 |86    | 96 |106 |116 |107 | 117|
+   * |--------------------------------------------------------'----`--------------|    |
+   * |100|110 |101 |      102             |112|103|113|104|114|105 |115 |118 |108 |    |
+   * `---------------------------------------------------------------------------------'
+   */
+
+
+#define LAYOUT( \
```

Comment:
Similarly to `h87a.h`, this macro should be renamed `LAYOUT_all`, and then `LAYOUT_100u_mods` and `LAYOUT_150u_mods` should only include the positions used by their physical layouts (104 and 101 keys, respectively).